examples/making-an-away-command.md

```diff
@@ -0,0 +1,120 @@
+# Making An Away Command
+Sometimes, when a user goes AFK, they get pinged, but the person who pinged them never knows the reason why. Creating an away command can be used to notify the user the reason or if someone is away when they get pinged.
+This is a greate and unique feature which can be implimented in a discord.js bot. If you follow this tutorial, you will successfully have an away command!
+
+## Creating The Command
+First of all, we will start by making a new Discord Collection
+```js
+const Discord = require('discord.js');
+const away = new Discord.Collection();
+```
+
+Now lets make the away command.
+```js
+const prefix = '/';
+if(message.content.startsWith(prefix + 'away')) {
+};

```

Comment:
You can't use semicolons in an if statement. And I believe we use a space between the if and the paren that holds the condition. Same applies for the rest of the file.

Reply:
where is the semicolon in the if statement?

Reply:
```javascript
if(message.content.startsWith(prefix + 'away')) {
};
```